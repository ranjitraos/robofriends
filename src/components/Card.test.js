import renderer from 'react-test-renderer';
import Card from './Card';

it('renders Card component correctly', () => {
    const card = renderer
        .create(<Card />)
        .toJSON();
    expect(card).toMatchSnapshot();
});