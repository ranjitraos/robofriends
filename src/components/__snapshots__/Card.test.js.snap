// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Card component correctly 1`] = `
<div
  className="tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5 card"
>
  <img
    alt="robot"
    height={200}
    src="https://robohash.org/undefined?size=200x200"
    width={200}
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
